src/java.base/share/classes/java/time/Instant.java

```diff
@@ -559,12 +559,12 @@ public ValueRange range(TemporalField field) {
     @Override  // override for Javadoc and performance
     public int get(TemporalField field) {
         if (field instanceof ChronoField) {
-            switch ((ChronoField) field) {
-                case NANO_OF_SECOND: return nanos;
-                case MICRO_OF_SECOND: return nanos / 1000;
-                case MILLI_OF_SECOND: return nanos / 1000_000;
-            }
-            throw new UnsupportedTemporalTypeException("Unsupported field: " + field);
+            return switch ((ChronoField) field) {
```

Comment:
Not really comment on the change itself, but Is this a leftover from the `instanceof` pattern variable exercise, or will we have another round for that? I see other locations which could be leftovers.

Reply:
Hi Naoto, I decided to only introduce the`instanceof` pattern variable where I thought it would add additional value to the code. In situations like this one, I thought there wasn't much point as the cast variable is only used once (in the switch). However, if you think I've overlooked something that would be beneficial to change, I'd be happy to take a look.

Reply:
I'd personally replace all the applicable locations, as otherwise, it would confuse why there are two idioms. But it is outside of this PR so probably for another day.

Reply:
I agree with Naoto that it's a bit strange.

Reply:
OK. If you guys think it's worthwhile. I've added in those changes now. Please see commit 3a1b716